exec/java-exec/src/main/java/org/apache/drill/exec/store/parquet/stat/ParquetFooterStatCollector.java

```diff
@@ -59,7 +59,7 @@ public ParquetFooterStatCollector(ParquetMetadata footer, int rowGroupIndex, Map
     // Reasons to pass implicit columns and their values:
     // 1. Differentiate implicit columns from regular non-exist columns. Implicit columns do not
     //    exist in parquet metadata. Without such knowledge, implicit columns is treated as non-exist
-    //    column.  A condition on non-exist column would lead to canDrop = true, which is not the
+    //    column.  A condition on non-exist column would lead to matches = true, which is not the
```

Comment:
I guess it is either ALL or NONE, but not `true`.

Reply:
ALL. Done